docs/user-guide/data-reduction.md

```diff
@@ -0,0 +1,141 @@
+# Reflectometry data reduction procedure
+
+The goal of the reflectometry data reduction is to compute the sample reflectivity $R(Q)$ as a function of the momentum transfer $Q$.
+
+## Preliminaries
+
+The detector data consists of a list $EV$ of neutron detector events.
+For each neutron event in the event list we know what wavelength $\lambda$ it had and we know the pixel number $j$ of the detector pixel that it hit.
+The detector pixel positions are known and so is the position of the sample and the orientation of the sample.
+From that information we can compute the reflection angle (assumed equal to the incidence angle) $\theta$, and the momentum transfer $Q$ caused by the interaction with the sample.
+
+The purpose of this text is not to describe how the event coordinates $Q$ and $\theta$ are derived from the raw event data and the geometry information, so for now just take those relations for given.
+
+To avoid overcomplicating the description it is assumed that the sample- and reference measurements were made over the same length of time, and it is assumed the neutron intensity from the source did not vary between the two measurements.
```

Comment:
Please explain that this only applies to the equations given here. The actual workflow has no such restriction.

Reply:
I think the actual workflow does actually currently have that restriction. Or am I mistaken?

Reply:
If it does, then it needs to change! But I guess that relates to #78.

Shouldn't this be easy to include in the maths? Isn't it just an extra factor in the normalisation? Something like $I_{\text{sample}} / I_{\text{ref}}$.

Reply:
> If it does, then it needs to change! 

Sure, that's what the issue is for. I've put it in the selected column on the dev board now.

Reply:
> Shouldn't this be easy to include in the maths? Isn't it just an extra factor in the normalisation? Something like 

Yeah probably, but I don't plan to include that in this PR. This is something we will have to iterate on as new features are added to the workflow.